swagger-ui/swagger.yaml

```diff
@@ -805,6 +847,73 @@
                     $ref: '#/definitions/CdpDepositResponse'
           500:
             description: Server internal error
+    /cdp/savings-rate-dist:
+      get:
+        summary: Get the total savings rate distributed by the cdp module
+        tags:
+          - CDP
+        produces:
+          - application/json
+        responses:
+          200:
+            description: The distributed savings rate
+              properties:
+                height:
+                  type: string
+                  example: "100"
+                result:
+                  savings_rate_distributed:
+                    type: string
+                    example: "5000000000000"
+          500:
+            description: Server internal error
+    /cdp/cdps:
+      get:
+        summary: Query all active cdps
+        tags:
+          - CDP
+        produces:
+          - application/json
+      parameters:
+          - in: query
+            name: owner
+            description: Owner address in bech32 format
+            required: false
+            type: string
+            x-example: kava1ffv7nhd3z6sych2qpqkk03ec6hzkmufy0r2s4c
+          - in: query
+            name: collateral-type
+            description: Collateral type
+            required: false
+            type: string
+            x-example: xrp-a
+          - in: query
+            name: id
+            description: CDP ID
+            required: false
+            type: string
+            x-example: "4"
+          - in: query
+            name: ratio
+            description: Collateralization ratio
+            required: false
+            type: string
+            x-example: "2.75"
+        responses:
+          200:
+            description: Query cdps
+            schema:
+              type: object
+              properties:
+                height:
+                  type: string
+                  example: "100"
+                result:
+                  type: array
+                  items:
+                    $ref: '#/definitions/CdpResponse'
+          500:
+            description: Internal Server Error
```

Comment:
At line 1397, incentive should be updated to collateral-type. So `{denom}` becomes `{collateral-type}` and the example becomes `Collateral Type` with example `bnb-a`

Reply:
https://github.com/Kava-Labs/kava/pull/653/commits/550065a767bc10dd230a8ee7f9a15a0b6742d1a3